Stripe/STPPaymentCardTextField.m

```diff
@@ -780,6 +783,23 @@ - (void)updateImageForFieldType:(STPCardFieldType)fieldType {
     }
 }
 
+- (NSString *)defaultCVCPlaceholder {
+    if (self.viewModel.brand == STPCardBrandAmex) {
+        return @"CVV";
+    } else {
+        return @"CVC";
+    }
+}
+
+- (void)updateCVCPlaceholder {
+    if (self.cvcPlaceholder) {
+        self.cvcField.placeholder = self.cvcPlaceholder;
+    } else {
+        self.cvcPlaceholder = [self defaultCVCPlaceholder];
+    }
+    self.cvcField.accessibilityLabel = [self defaultCVCPlaceholder];
```

Comment:
Should we be setting the accessibility label to your custom placeholder as well? Or are you intentionally always setting it to the default?